### ISSUE TITLE
support for single use feedreading (with or without ini)

### ISSUE THREAD

**Issue by infominer33** [feedreader/pluto#34]
Sometimes I want to print one or two feeds onto a single page so I can review them\for research.

I made a "try.ini" to re-use for these cases, but realizing I need to add a line to my make-file so it deletes the existing database before running the feed, otherwise I get posts from previous runs mixed in.

Wondering if you'd consider enabling that by commandline, something like:

`pluto b "https://identosphere.substack.com/feed/, https://decentralized-id.com/feed.xml" -t planetid -o try/`

**Comment by geraldb** [comment]
Good point. Great idea.  +1 - I have planned for a long time and actually started early this year to add more configuration formats including just a list of feeds (no .ini - just text) and the "standard / classic" opml.     

About testing I guess you want to test the merge of the templates / page generation - otherwise if you want to test the feed you can use the heart of pluto, that is, the feedparser in a little script, see <https://github.com/feedparser/feedparser#read-feed-example>.

Maybe make a list of some goodies for 2021 and the next pluto update:
- also high on the list is support for tags / categories - from the feed and custom
- and better support for (multi) author info from the feed
- and your idea - easier configuration (just "plain" feed lists) and testing of merging of templates

Thanks again. Keep it up.

**Comment by infominer33** [comment]
love it!  mostly my single-use case is research for content creation \ link collection... or even to see if I already wrote about something.. I just like looking at the entire blog on one page, then can even import to notion and then export to markdown the whole thing for making curated lists and sharing about the history of an org\company
